### PR TITLE
Suhrid branch

### PR DESCRIPTION
**Pull request by AosafRahman**
comment on my code

### CODE REVIEW

**Comment by LilMarz** [comment]
Okay have a nice day bye bye have a great time

File: calculator.py [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+def multiply():
+    print("What is your first number?")
+    num1 = int(input())
+    print("What is your second number?")
+    num2 = int(input())
+    answer = num1 * num2
+    return answer
+
+print("Your final answer is", multiply())
```
Comment (Oozegy) [inline calculator.py:33]: good enough

**Review by LilMarz — APPROVED** [review]
Good job you did well. SIKE

**Review by IanRBr — APPROVED** [review]
The code is good, but you know you need to take away the return answer and

**Review by LilMarz — APPROVED** [review]
goodz

**Review by Oozegy — APPROVED** [review]

**Review by IanRBr — APPROVED** [review]
this is good

**Pull request merged**